src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static Colors getRandomColor() {
+
+        Random random = new Random();
```

Comment:
move Random to the class field, like: 
![image](https://user-images.githubusercontent.com/12864128/220933042-8f4b3256-9421-499c-b08a-a937f2203930.png)